### DISCUSSION TITLE
Pattern Order of Operation overiden

### DISCUSSION THREAD

**Discussion by Tony-Lee-H** [TimPaterson/Fusion360-Batch-Post#36] [Q&A] [answered]
Thank you for Post Process All.
When posting a group of operations as a pattern, the 'Operation order' selection always defaults to 'Order by operation'. Selecting 'Preserve order' or 'Order by tool' gets overridden. 
Appreciate any help.

![image](https://user-images.githubusercontent.com/95996204/145702996-5631dfd5-f035-4a56-90c1-9fe77dc8c858.png)

**Comment by TimPaterson** [comment, marked as answer]
PostProcessAll asks for a list of operations from the [Fusion 360 API](https://help.autodesk.com/view/fusion360/ENU/?guid=GUID-7B5A90C8-E94C-48DA-B16B-430729B734DC). It seems that Fusion 360 always returns the list in "Order by operation", ignoring the Operation Order selection. I don't know of any way to work around this.

**Reply by Tony-Lee-H** [reply]
Oh, that's unfortunate. Wonder if they may update their API at some stage. Fingers crossed.
